docs/how-to/advanced/enable-archs.md

```diff
@@ -0,0 +1,52 @@
+---
+tags: [how-to, advanced]
+---
+
+# How to enable additional architectures
+
+By default, feedstocks are created with support for the main platforms (e.g. `linux-64`, `osx-64`, `win-64`). New architectures can be added by changing the default configuration in `conda-forge.yml` and rerendering. However, since building a new architecture requires that your dependencies are also available for that architecture, this is usually controlled by migrations triggered by changes on specific files available in the `conda-forge-pinning-feedstock` repository.
+
+Before submitting a build request for a new architecure, check whether:
+
+1. The package is already built for that architecture. Consult the package page in anaconda.org: `https://anaconda.org/conda-forge/<package-name>`.
+2. Your package is already in the process of being migrated in the [corresponding migration status page](https://conda-forge.org/status/).
+3. The feedstock in question has already an issue or pull request for that architecture, which may contain information why the package is not yet available.
+
+If you do see an open PR (or once a PR is submitted by the migration), you can help the feedstock maintainers make the build pass. If you have insight into the particular package, **please** chime in, but most of all **be patient and polite**.
+
+Once the new builds are available from `anaconda.org`, please help the maintainers by testing the packages, and reporting back with any problems… but also successes!
+
+## General steps
+
+Adding a new architecture usually means following the same steps, but targeting different configuration files:
+
+1. Fork `conda-forge-pinning-feedstock` and create a fresh new branch. See [How to fork your feedstock](../basics/fork-sync.md) for more details.
+2. Edit the migration file for your architecture (see [architecture-specific information](#architecture-specific-information)) to add the name of your feedstock, sans the `-feedstock` suffix. Make sure to insert it in the right position to respect alphabetical order, or run `pre-commit run -a`.
```

Comment:
I like French as much as the next guy, but it doesn't really help with accessibility here IMO
```suggestion
2. Edit the migration file for your architecture (see [architecture-specific information](#architecture-specific-information)) to add the name of your feedstock, without the `-feedstock` suffix. Make sure to insert it in the right position to respect alphabetical order, or run `pre-commit run -a`.
```